Code/GraphMol/ChemReactions/Wrap/rdChemReactions.cpp

```diff
@@ -211,6 +211,22 @@ void RemoveUnmappedProductTemplates(ChemicalReaction *self,
   }
 }
 
+void RemoveAgentTemplates(ChemicalReaction *self, python::object targetList){
```

Comment:
needs a `PRECONDITION(self,"no reaction")` or just switch it to `self` being a reference (should work fine in the wrappers).